src/bin/stratis_uuids_to_names.rs

```diff
@@ -42,8 +42,10 @@ const STRATIS_MANAGER_OBJECT: &str = "/org/storage/stratis2";
 const STRATIS_POOL_IFACE: &str = "org.storage.stratis2.pool.r1";
 const STRATIS_FS_IFACE: &str = "org.storage.stratis2.filesystem";
 const DBUS_OM_IFACE: &str = "org.freedesktop.DBus.ObjectManager";
+
 lazy_static! {
-    static ref TIMEOUT: Duration = Duration::new(5, 0);
+    // The value chosen is the maximum timeout permitted for the Stratis CLI
+    static ref TIMEOUT: Duration = Duration::new(1_073_741, 823);
```

Comment:
This is a very magic number, and stratis-cli also doesn't explain it. In my experience, there is no maximum timeout with regard to D-Bus method calls and it is totally fine to just wait forever (or u64::MAX seconds).

Moreover, stratis-cli has ` maximum_dbus_timeout_ms = 1073741823`, but the second argument to `Duration::new` is nanoseconds. Thus, this should be `Duration::new(1_073_741, 823_000_000)`, no?

Reply:
Probably hex makes it more obvious: 0x3fffffff = 1073741823, 2^30 - 1. The _reason_ the exponent was chosen to be 30 has much to do with the mechanics of floating point conversion; it was the largest integer value that could be guaranteed to be transmitted without its value being changed via numeric conversions through the Python bindings that we use in the CLI to the underlying C library w/out our doing sophisticated analysis of floating point conversions (which seemed a waste of time). But there is no floating point conversion in the Rust source! So we can do something simpler.

Reply:
Looks like we can be direct and use the maximum value that can be stored in a 32 bit signed int.

Reply:
We can't play it safe and use the maximum value that can be stored in a 16-bit unsigned int, that is way too small for practical purposes.

Reply:
@mulkieran Just to make sure I understand, the value `1073741823` is in milliseconds because that's what D-Bus ultimately takes if I remember correctly, right? If so, `823` as the 2nd parameter will not yield that value because that's 823 nanoseconds. This would result in 1,073,741.000000823 seconds or 1,073,741,000.000823 milliseconds. Am I missing something here?

Reply:
Right, as we can see [here](https://docs.rs/dbus/0.8.4/src/dbus/channel.rs.html#269-279), the dbus crate uses `Duration::as_millis` to convert the timeout Duration object to the integer value used with libdbus.

libdbus treats i32::MAX as infinite as documented [here](https://dbus.freedesktop.org/doc/api/html/group__DBusPendingCall.html#ga21384c9c5b0da54f7d0a92012522f213). So I think this deserves a named constant in the dbus crate, but the second best thing would be a named constant in stratisd.

Reply:
We should probably prefer to be good citizens and submit to the upstream crate...